build.xml

```diff
@@ -119,11 +120,22 @@
         </condition>
     </target>
 	
-	<target name="jarSkipTests">
+	<target name="jarSkipTests" description="create jar file skip unit tests">
 		 <property name="RUN_NO_TESTS" value="Yes"/>
 		<antcall target="jar"/>
 	</target>
 
+	<target name="jarSkipTestAndSubprojectBulds" description="create jar file skip unit tests do not build sub projects">
+		 <property name="RUN_NO_TESTS" value="Yes"/>
```

Comment:
Please correct the spelling in the target name:   "Builds", not "Bulds".

Reply:
pushed fix.